features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/.eslintignore

```diff
@@ -9,6 +9,7 @@ source/src/app/data/*
 /source/src/app/components/Apis/Details/Comments/*
 /source/src/app/components/Apis/Details/Credentials/*
 /source/src/app/components/Apis/Details/Documents/*
+/source/src/app/components/Apis/Details/GraphQLConsole/*
```

Comment:
Shall we fix any eslit issues found in the components?  We also need to enable other components and fix them as well.